CHANGELOG.rst

```diff
@@ -5,12 +5,12 @@ Changelog
 x.x.x (TBD)
 ===================
 
-* Added ...
+* Added support for colors in stat mapping panel with StatValueMappings & StatRangeMappings
 
 Changes
 -------
 
-* Changed ...
+* Deprecated StatMapping, StatValueMapping & StatRangeMapping
```

Comment:
Where does the deprecation happen?

Reply:
Maybe deprecation is not the best phase, Grafana v8 now has a new scheme for mappings so I have just added a note to the docstring for those Classes. 
```suggestion
* Deprecation warning StatMapping, StatValueMapping & StatRangeMapping when using Grafana v8
```

Reply:
Got it, thanks!